src/phpDocumentor/Descriptor/Builder/PhpParser/MethodAssembler.php

```diff
@@ -45,11 +45,11 @@ public function __construct(ArgumentAssembler $argumentAssembler)
      *
      * @param ClassMethod $data
      *
-     * @return MethodDescriptor
+     * @return Method
      */
     public function create($data)
     {
-        $methodDescriptor = new MethodDescriptor();
+        $methodDescriptor = new Method();
```

Comment:
I will have to rewrite the Assemblers in to factory strategies, So this will be removed in a next pr.